stack.yaml

```diff
@@ -0,0 +1,8 @@
+resolver: nightly-2022-07-23
+
+extra-deps:
+  - github: uhbif19/colourista
+    commit: 8148a0446bf61814f79d6a0c497007fde72b31eb
+  - hspec-2.10.0@sha256:2d1dc53e361998f002497b7222fdb700cc540b2aaf880e2619a0b5d54abb74e5,1712
+  - hspec-core-2.10.0@sha256:a5d7d3e7bbcf42e8def370d86ab08252821be6fb9d00dddb3c484973d7a6e801,6617
+  - hspec-discover-2.10.0@sha256:517d0892217539969c199f9cc30065db7bcd68ff5512095e4f94ff8ef35d1313,2166
```

Comment:
I see that this snapshot contains `hspec*` libraries but with the version `2.9.7`

Let's remove these 3 lines from `stack.yaml` and relax the lower bound for `hspec` in our test suite to allow version `2.9.7`
https://github.com/chshersh/iris/blob/e79fdc2bebc1caf58e4b566a678db4392929e355/iris.cabal#L127

I expect that our project should still build and work with the lower version of `hspec` but this way we'll reduce the `extra-deps` burden for `stack`